adapters/base/src/main/java/org/apache/cassandra/sidecar/adapters/base/CassandraAdapter.java

```diff
@@ -107,30 +99,7 @@ public Metadata metadata()
     @Nullable
     public NodeSettings nodeSettings()
     {
-        ResultSet rs = executeLocal("SELECT "
-                                              + RELEASE_VERSION_COLUMN_NAME + ", "
-                                              + PARTITIONER_COLUMN_NAME + ", "
-                                              + DATA_CENTER_COLUMN_NAME + ", "
-                                              + RPC_ADDRESS_COLUMN_NAME + ", "
-                                              + RPC_PORT_COLUMN_NAME + ", "
-                                              + TOKENS_COLUMN_NAME
-                                              + " FROM system.local");
-        if (rs == null)
-        {
-            return null;
-        }
-
-        Row oneResult = rs.one();
-
-        return NodeSettings.builder()
-                           .releaseVersion(oneResult.getString(RELEASE_VERSION_COLUMN_NAME))
-                           .partitioner(oneResult.getString(PARTITIONER_COLUMN_NAME))
-                           .sidecarVersion(sidecarVersion)
-                           .datacenter(oneResult.getString(DATA_CENTER_COLUMN_NAME))
-                           .tokens(oneResult.getSet(TOKENS_COLUMN_NAME, String.class))
-                           .rpcAddress(oneResult.getInet(RPC_ADDRESS_COLUMN_NAME))
-                           .rpcPort(oneResult.getInt(RPC_PORT_COLUMN_NAME))
-                           .build();
+        throw new UnsupportedOperationException("Node settings are not provided by this adapter");
```

Comment:
`CassandraAdapterDelegate` manages its own node settings object, and never delegates the call to this method. We remove the implementation here for simplicity